frontend/src/components/UserSettings/UserSettings.js

```diff
@@ -101,8 +101,7 @@ class UserSettings extends React.Component {
     email: '',
     first_name: '',
     last_name: '',
-    phone: '1',
```

Comment:
The '1' is in the phone number as a default if nothing is there. Otherwise the format of the field  looks broken. It doesn't need to be a '1', but we should have something there by default to make sure the formatting of the field looks good when there is no phone number coming from the database.